mix.exs

```diff
@@ -113,8 +113,9 @@ defmodule EMQXUmbrella.MixProject do
     set_emqx_app_system_env(apps, profile_info, version)
   end
 
-  defp umbrella_apps(profile_info) do
-    enterprise_apps = enterprise_umbrella_apps()
+  defp umbrella_apps(profile_info = %{release_type: release_type}) do
+    enterprise_apps = enterprise_umbrella_apps(release_type)
+    excluded_apps = excluded_apps(release_type)
```

Comment:
Nit: I think this is not needed here, since we enumerate the umbrella apps under `apps/*`, and the excluded apps are external ones such as `quicer`, `observer`, etc. .

Reply:
Yes, correct. I added this to reduce number of different lines in mix.exs between this and what we have in the private fork.